graphql/ui_plugin_build_baron.go

```diff
@@ -115,21 +115,35 @@ func BbGetCreatedTicketsPointers(taskId string) ([]*thirdparty.JiraTicket, error
 	return results, nil
 }
 
-func GetSearchReturnInfo(taskId string, exec string) (*thirdparty.SearchReturnInfo, bool, error) {
-	projectNotFoundError := false
+type buildBaronConfig struct {
+	ProjectFound     bool
+	SearchConfigured bool
+}
+
+func GetSearchReturnInfo(taskId string, exec string) (*thirdparty.SearchReturnInfo, buildBaronConfig, error) {
+	bbConfig := &buildBaronConfig{}
 	t, err := BbGetTask(taskId, exec)
 	if err != nil {
-		return nil, projectNotFoundError, err
+		return nil, *bbConfig, err
 	}
 	settings := evergreen.GetEnvironment().Settings()
 	buildBaronProjects := BbGetConfig(settings)
 	bbProj, ok := buildBaronProjects[t.Project]
 
 	if !ok {
-		projectNotFoundError = true
-		return nil, projectNotFoundError, errors.New(fmt.Sprintf("Build Baron project for %s not found", t.Project))
+		bbConfig.ProjectFound = false
+		return nil, *bbConfig, errors.New(fmt.Sprintf("Build Baron project for %s not found", t.Project))
 	}
+	bbConfig.ProjectFound = true
 
+	// the build baron is configured if the jira search is configured
+	bbSearchConfigured := len(bbProj.TicketSearchProjects) > 0
```

Comment:
you aren't using this variable so you may as well check the condition at line 142 (the comment you already have is sufficient to explain it)